ccw.core/plugin.properties

```diff
@@ -153,3 +153,5 @@ ccw.leiningen.problemmarker.classpathcontainer=Leiningen Classpath Container
 ccw.leiningen.command.launch-headless-repl.name=Launch Headless REPL for the project
 ccw.leiningen.command.launch-headless-repl.description=Launch a Headless REPL for the project through Leiningen
 
+Dummy.label=This dummy label is dedicated to all the contributors of org.eclipse.jdt.
+Presentation.label= Clojure

```

Comment:
The presentation is for the extension, Dummy...well I think nothing in this patch, it was probably used in the hovers...I will remove it.